crates/logging/src/tracing_logger.rs

```diff
@@ -30,9 +30,17 @@ impl Logger for TracingLogger {
         level: LogLevel,
         add_source: bool,
     ) -> Self {
+        let tracing_level = match level {
+            LogLevel::Error => tracing::Level::ERROR,
+            LogLevel::Warn => tracing::Level::WARN,
+            LogLevel::Info => tracing::Level::INFO,
+            LogLevel::Debug => tracing::Level::DEBUG,
+            LogLevel::Trace => tracing::Level::TRACE,
+            LogLevel::Fatal => tracing::Level::ERROR, // `Fatal` can map to `ERROR` or a custom level if supported
```

Comment:
can can you put the Fatal logger at the top? also could you add custom processing so that the Fatal level is indeed fatal (i.e. panics)